tests.py

```diff
@@ -311,6 +311,28 @@ def test_import_from_qtwidgets():
     assert QPushButton.__name__ == "QPushButton", QPushButton
 
 
+def test_pyside2uic():
```

Comment:
Wait, what are we testing here? Are we testing Qt.py, or are we testing pyside2-tools?

We only need to test what we add to the table. In this case, `python -m Qt --compile my_ui.ui` and the `--convert` option.